Dockerfile

```diff
@@ -1,25 +1,47 @@
-FROM registry.access.redhat.com/ubi8/python-39 as base
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS build
 
 # Add application sources to a directory that the assemble script expects them
 # and set permissions so that the container runs without root access
 USER 0
 
-RUN dnf -y remove httpd-tools httpd-filesystem
+RUN microdnf install -y python39-devel postgresql-devel gcc && \
+    pip3 install virtualenv                                 && \
+    mkdir -p /opt/app-root                                  && \
+    chown 1001:0 /opt/app-root
```

Comment:
I think we could keep `python-39` as the *build*  image, it is my understanding there's no usage in prod for it as we use this multi-stage build as we end up using the minimal. 

That way we'd just have to install `postgresql-devel` and `gcc`.

Reply:
I am building constantly this on my computer, I don't like the idea of having one extra base image. Also I don't know how the safety scan works with prodsec so I'd like to play it safe.